yaml_in/providers/cloud-simulators.yaml

```diff
@@ -0,0 +1,24 @@
+id: cloud_simulatorsId
+title: Cloud Simulators
+description: Cloud simulators are cloud services that allow you to simulate quantum programs in high performance computers.
```

Comment:
Cloud services that allow you to simulate quantum programs in high performance computers.

Reply:
@1ucian0 I think you wanted to avoid repeating the mention of category name in the description, right? I fixed it here: https://github.com/HuangJunye/qiskit-providers-quickstart/commit/fd31d572d765953554aa8cedd526b230223b9f3e